docs/instructlab-cli-1.0.0.md

```diff
@@ -0,0 +1,35 @@
+# The Road to 1.0.0
```

Comment:
I'm not anxious to push for a 1.0.0 release for the sake of **having** a >1.0.0 project. I don't personally believe that semver major versions are all that meaningful to the end user. 

However, these are good objectives for us and would represent a major change in the implementation and design of the project, which I'm certainly on board with.